bazel/repositories.bzl

```diff
@@ -91,19 +91,27 @@ cc_library(
     visibility = ["//visibility:public"],
 )
 
+# There is some moderate evil here. buildconf needs to be invoked in its source
+# location, where it also generates temp files (no option to place in TMPDIR).
+# Also, configure does a cd relative dance that doesn't play nice with the
+# symlink execroot in the Bazel build. So, copy everything prior to building.
+ARES_CONFIG_SH_CMDS = [
+    "export TMPDIR=$$(realpath $(@D))",
+    "rm -rf $(@D)/cares_build",
+    "mkdir -p $(@D)/cares_build",
+    "rsync -aRLv $(SRCS) $(@D)/cares_build",
```

Comment:
I would not assuming `rsync` available on the machine, can't this be done by `cp` with `-L`?